libcontainer/utils/utils_test.go

```diff
@@ -23,3 +29,101 @@ func TestGenerateName(t *testing.T) {
 		t.Fatalf("expected name to be %d chars but received %d", expected, len(name))
 	}
 }
+
+func TestResolveRootfs(t *testing.T) {
+	dir := "rootfs"
+	os.Mkdir(dir, 0600)
+	defer os.Remove(dir)
+
+	path, err := ResolveRootfs(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+	pwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if path != fmt.Sprintf("%s/%s", pwd, "rootfs") {
+		t.Errorf("expected rootfs to be abs and was %s", path)
+	}
+}
+
+func TestResolveRootfsWithSymlinc(t *testing.T) {
+	dir := "rootfs"
+	os.Symlink("/var", dir)
```

Comment:
This test assumes that we have to be root to run the test suite, right? Why not set up some directory in `/tmp` where you make the symlink.